doc/bt/CZ_RUN_GAMEEXIT_TIMER.bt

```diff
@@ -0,0 +1,18 @@
+//------------------------------------------------
+//--- 010 Editor v8.0 Binary Template
+//
+//      File: CZ_RUN_GAMEEXIT_TIMER.bt
+//   Authors: celophi
+//   Version: i170175
+//   Purpose: Notifies the server that the client wants to make a state transition.
+//  Category: 
+// File Mask: 
+//  ID Bytes: 
+//   History: 
+//------------------------------------------------
+
+short op;
+int sequence;
+int checksum;
+
+string destination; // Maximum length is 32 characters.
```

Comment:
If this is a fixed size string, shouldn't it be char[32]?

Reply:
It's not fixed. If you write it as `char[32]`, the array will contain junk bytes. (The value of the column displayed will be correct though).

Reply:
It's not? It's always 32 byte, isn't that the definition of fixed?^^ You're right that the array will contain junk the client didn't clear of course, but what will you do if, for example, they decide to add an int after destination now?

Reply:
Oh I see. Perhaps I didn't realize what "fixed" means.

Reply:
At least that's what I meant^^